### PR TITLE
Add CampaignDetailPage

### PR DESCRIPTION
**Pull request by toshiakitakase**
- Show campaign info
- Can cancel campaign
- Can refund from campaign

WIP: Audius integration

### CODE REVIEW

**Comment by wild-mouse** [comment]
Found typo: `CampaignDetailPageTem"apl"ate/index.tsx` => `CampaignDetailPageTemplate/index.tsx`

File: frontend/packages/react-app/package.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
     "ethers": "^5.0.17",
     "graphql": "^14.6.0",
     "ipfs-http-client": "^48.1.3",
+    "moment": "^2.29.1",
```
Comment (wild-mouse) [inline frontend/packages/react-app/package.json:68]: momentjs is treated as legacy by themselves. 
https://momentjs.com/docs/#/-project-status/

You can use other recommendations.
https://momentjs.com/docs/#/-project-status/recommendations/

**Comment by wild-mouse** [comment]
After campaign creation, detail page shows "End campaign" button, but campaign is not started yet and so it should be shows "Cancel campaign" button instead. 
Screen Shot 2020-12-22 at 12 24 15

File: frontend/packages/react-app/package.json (hunk 1 above)
Comment (toshiakitakase) [reply frontend/packages/react-app/package.json]: Fixed

**Review by wild-mouse — APPROVED** [review]
LGTM 🏃‍♀️

**Pull request merged**